### ISSUE TITLE
Excel/CSV export of table

### ISSUE THREAD

**Issue by Wolfsblvt** [Wolfsblvt/BlossomsPokemonGoManager#63]
That shouldn't be such a difficult task, but it would be cool.

Just one button that exports the table as CSV, opens a file dialog and lets the user save it.

**Comment by Wolfsblvt** [comment]
I'll take that soon, I guess.

**Comment by JaymerJaymer** [comment]
It already can copy/paste into a SS, as is

**Comment by weblue** [comment]
you can copy the information directly from the table into Excel or Google Sheets; is this really necessary?

**Comment by weblue** [comment]
@Wolfsblvt thoughts?

**Comment by Wolfsblvt** [comment]
@weblue It's really small part of code and it's hidden in the File menu I would say. So is there a reason to not include that, just so you have a direct export? :D

**Comment by Lacero** [comment]
This would be a helpful feature. It's inconvenient how the column headers cannot be copied along with the pokemon inventory. If not going to implement, could add a "Ctrl+A to Select All" option under the File menu, if Windows OS is detected, for those who are unaware of the shortcut.

**Comment by BHMath** [comment]
I agree copying all lines work fine but headers are missing and this is annoying.

**Comment by cyan** [comment]
i would say adding the ability to dump those data automatically at start would be nice ^^

**Comment by tdshimo** [comment]
Oh, I see, copy-n-paste _does_ work with Excel. Kinda. :) That's fine for now! No rush on .csv from my perspective.

**Comment by cleka** [comment]
Yeah, took also me a while to realize one can copy-paste, was searching for export feature too. 
So to the very least perhaps should add a menu item "File=>Export" which tells about the copy paste way. And perhaps puts it ready to clipboard...

**Comment by weblue** [comment]
I'm glad you had a happy ending tho

**Comment by haferje** [comment]
i would like an option to export as JSON.  it is much easier to paste into a file and run some JS logic against it.

**Comment by Wolfsblvt** [comment]
Thanks for reporting this and for using BlossomsPokemonGoManager. This project is no longer maintained and no longer works with current Pokémon Go services, so I’m closing all remaining issues as part of archiving it. Please do not use it with a live account or current credentials. See the [Disclaimer](https://github.com/Wolfsblvt/BlossomsPokemonGoManager#disclaimer) for details.

**Comment by Wolfsblvt** [comment]
Thanks for reporting this and for using the project. Blossoms Pokémon Go Manager is no longer maintained and no longer works with current Pokémon Go services, so I’m closing all remaining issues as part of archiving the repository. This avoids leaving a false impression that further development or support is planned.
